project/plugins.sbt

```diff
@@ -1,9 +1,11 @@
 resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc"      %  "sbt-auto-build"               % "3.9.0")
-addSbtPlugin("org.scoverage"    %  "sbt-scoverage"                % "1.9.3")
-addSbtPlugin("org.scalameta"    %  "sbt-scalafmt"                 % "2.4.6")
-addSbtPlugin("ch.epfl.scala"    %  "sbt-scalafix"                 % "0.10.2")
-addSbtPlugin("org.scalastyle"   %% "scalastyle-sbt-plugin"        % "1.0.0")
-addSbtPlugin("ch.epfl.scala"    %  "sbt-bloop"                    % "1.5.3")
+addSbtPlugin("uk.gov.hmrc"     % "sbt-auto-build"        % "3.20.0")
+addSbtPlugin("org.scoverage"   % "sbt-scoverage"         % "2.0.9")
+addSbtPlugin("org.scalameta"   % "sbt-scalafmt"          % "2.5.2")
+addSbtPlugin("ch.epfl.scala"   % "sbt-scalafix"          % "0.11.1")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("ch.epfl.scala"   % "sbt-bloop"             % "1.5.11")
```

Comment:
And 1.5.13 but we should look to get rid completely if we can get all vscode devs to set it globally.